storybook/storybook-react/config/preview.tsx

```diff
@@ -3,6 +3,8 @@ import '@amsterdam/design-system-assets/font/index.css'
 import '@amsterdam/design-system-css/dist/index.css'
 import { viewports } from './viewports'
 
+import './story.css'
```

Comment:
We currently use `storybook-react/config/preview-body.html` for story-only styling, this can go there. (@VincentSmedinga: why did we move this from a css file to a html file with style tags?)

Reply:
I follow [storybook documentation](https://storybook.js.org/docs/react/configure/styling-and-css), I could move the preview-body css to this file.

Reply:
I would do that for now, we'll figure out how we want to implement story-only styling later.